models/profiles.yaml

```diff
@@ -59,253 +81,6 @@ models:
-            dependencies:
-              - session_start_time
-        - entity_var:
-            name: last_seen_date
-            from: models/rsTracksUnionPages
-            select: max(date(session_end_time))
-            description: The latest date on which an event has been recorded by the user
-            dependencies:
-              - session_end_time
-        - entity_var:
-            name: first_campaign_name
-            from: models/rsPages
-            select: first_value(context_campaign_name)
-            window:
-              order_by:
-                - timestamp asc
-            where: context_campaign_name is not null and context_campaign_name!=''
-            description: First campaign from which a user has visited the app
-        - entity_var:
-            name: last_campaign_name
-            from: models/rsPages
-            select: last_value(context_campaign_name)
-            window:
-              order_by:
-                - timestamp asc
-            where: context_campaign_name is not null and context_campaign_name!=''
-            description: Latest campaign from which a user has visited the app
-        - entity_var:
-            name: first_source_name
-            from: models/rsPages
-            select: first_value(context_campaign_source)
-            window:
-              order_by:
-                - timestamp asc
-            where: context_campaign_source is not null and context_campaign_source!=''
-            description: First source from which a user has visited the app
-        - entity_var:
-            name: last_source_name
-            from: models/rsPages
-            select: last_value(context_campaign_source)
-            window:
-              order_by:
-                - timestamp asc
-            where: context_campaign_source is not null and context_campaign_source!=''
-            description: Last source from which a user has visited the app
-        - entity_var:
-            name: campaigns_list
-            from: models/pages_orderby_table
-            select: array_agg( context_campaign_name)
-            description: list of all campaigns from which a user has visited the app, sorted in chronological order, from oldest to newest
-        - entity_var:
-            name: mediums_list
-            from: models/pages_orderby_table
-            select: array_agg( context_campaign_medium)
-            description: list of all mediums from which a user has visited the app, sorted in chronological order, from oldest to newest
-        - entity_var:
-            name: sources_list
-            from: models/pages_orderby_table
-            select: array_agg( context_campaign_source)
-            description: list of all sources from which a user has visited the app, sorted in chronological order, from oldest to newest
-            dependencies:
-              - session_start_time
       features:
         - days_since_last_seen
         - days_since_account_creation
```

Comment:
@am6010 @debanjan97 Do we not need to remove this from here if it is removed above ?

Reply:
probably yes
all the removing/ adjusting was done by migration scripts, auto migration might have skipped it

Reply:
removed